rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -1360,7 +1360,7 @@ private MinerClock getMinerClock() {
         return new IndexedBlockStore(blockFactory, indexMap, blocksDB, indexDB);
     }
 
-    private static KeyValueDataSource makeDataSource(String name, String databaseDir) {
+    public static KeyValueDataSource makeDataSource(String name, String databaseDir) {
```

Comment:
In another PR we should remove static functions from RskContext

Reply:
Agreed